demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2030,31 +1982,24 @@ export class DemoMeetingApp
     // Hang on to this so we can unsubscribe later.
     this.activeSpeakerHandler = (attendeeIds: string[]): void => {
       // First reset all roster active speaker information
-      for (const attendeeId in this.roster) {
-        this.roster[attendeeId].active = false;
+      for (const id of this.roster.getAllAttendeeIds()) {
```

Comment:
Should this just be move to the roster component itself (in an API calls update active speaker or something? Its weird that the caller of the component has to do all this :)

Reply:
Makes sense. I can move it to the roster component.

Reply:
While trying to move the logic to the roster component, I realized that part of the work in this section is being done to update the roster and some part is done to update the `videoTileCollection`. At this point I will like to keep this section as is.